model/requirements.txt

```diff
@@ -0,0 +1,81 @@
+absl-py==2.1.0
```

Comment:
We also need to exclude this all requirements.txt files while building the pip package. The current implementation of the wheel build includes all requirements.txt files within the toolkit repository. I think we can avoid including these requirements in the Python wheel?